packages/math-input/src/types.ts

```diff
@@ -40,11 +40,11 @@ export type Icon = {
 };
 
 export type KeyConfig = {
-    ariaLabel: string;
     id: Key;
     type: keyof typeof KeyTypes;
-    childKeyIds: Array<Key>;
-    icon: Icon;
+    childKeyIds: ReadonlyArray<string>;
+    ariaLabel?: string;
+    icon?: Icon;
```

Comment:
I don't like making these things optional or moving from `Key` to `string`, but according to the code these are more correct.

Reply:
Are we missing `icon` and `ariaLabel` in a few places?

Reply:
Only in a couple of places. Trying my best to type without editing logic right now. Hopefully the typing I've done since this has made it a little clearer.

``` JSX
            const keyConfig = {
                id: Keys.MANY,
                type: KeyTypes.MANY,
                childKeyIds: keys,
            };
            return <TouchableKeypadButton keyConfig={keyConfig} {...rest} />;
```